app/assets/stylesheets/_card.scss

```diff
@@ -0,0 +1,89 @@
+.card {
+  border-radius: 0;
+  box-shadow: 0 1px 1px rgba(0,0,0,0.1);
+  border: 0;
+
+  &.clickable {
+    cursor: pointer;
+  }
+
+  .panel-heading {
+    background-color: inherit;
+    font-weight: bold;
+    color: $darkgrey-2;
+    border-bottom: 1px solid $lightgrey-11;
+
+    .mi {
```

Comment:
We should not add behavior to classes coming from a library, if eventually the library change the name of the class and we change it on the view it'll stop working. Add a new class for that 😄